package.json

```diff
@@ -128,7 +128,7 @@
       "lint": {},
       "lint-list-item-indent": "mixed",
       "lint-maximum-line-length": 80,
-      "lint-no-missing-blank-lines": true,
+      "lint-no-missing-blank-lines": false,
```

Comment:
this being set to true returns a false positive lint error for nested markdown lists

Reply:
@dakotablair you enabled this, so you may want to review its disabling

Reply:
Hmm, changing the spacing didn't work for me (it then complained about incorrect indent levels)